_sources/UsingRESTAPIs/cachingResponses.rst

```diff
@@ -15,120 +15,83 @@ You haven't experienced it yet, but if you get complicated data back from a REST
 
 To avoid re-requesting the same data, we will use a programming pattern known as **caching**. It works like this:
 
-1. Before doing some expensive operation (like calling ``requests.get`` to get data from a REST API), check whether your cache contains the results that would be generated by that operation.
-2. If so, return the cached results.
-3. If not, perform the expensive operation and save the results (e.g. the complicated data) in your cache so you won't have to perform it again th enext time.
+1. Before doing some expensive operation (like calling ``requests.get`` to get data from a REST API), check whether you have already saved contains the results that would be generated by making that request.
+2. If so, return that same data
+3. If not, perform the expensive operation and save the results (e.g. the complicated data) in your cache so you won't have to perform it again the next time.
```

Comment:
Not really, but trying to provide a lay-terms explanation of the whole process before introducing the idea that this thing they already understand is what "caching" is, that the saved info is the "cache". 

I'll try another way, or include the word along with the explanation.